src/base/server_type.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2021 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "server_type.hpp"
+
+#include "system_parameter.h"
+
+static SERVER_TYPE server_type;
```

Comment:
Rename to make global variable obvious, by adding 'g_' or 'global_' prefix or by naming as server_Type.